Momento/Responses/CacheMultiGetFailureResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace MomentoSdk.Responses
+{
+    public class CacheMultiGetFailureResponse
+    {
+        public byte[] Key { get; private set; }
+        public Exception Failure { get; private set; }
+
+        public CacheMultiGetFailureResponse(byte[] key, Exception faiure)
```

Comment:
Second argument has a typo: `faiure` should be `failure`
```suggestion
        public CacheMultiGetFailureResponse(byte[] key, Exception failure)
```